src/common/cis2pi.jl

```
@@ -1,5 +1,15 @@
 """Function to compute exp(2π i x)"""
 cis2pi(x) = cispi(2x)
```

Is this needed at all. I mean could this not just be folded into the definition below ?

---

Yes, the special casing is incorrect for ForwardDiff (it will silently ignore the dual part)